LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/parsing/background-image-computed.sub-expected.txt

```diff
@@ -2,35 +2,35 @@
 PASS Property background-image value 'radial-gradient(at 10px 10px, rgb(255, 0, 0), rgb(0, 0, 255))'
 PASS Property background-image value 'radial-gradient(farthest-side, rgb(255, 0, 0), rgb(0, 0, 255))'
 PASS Property background-image value 'radial-gradient(farthest-side at 10px 10px, rgb(255, 0, 0), rgb(0, 0, 255))'
-FAIL Property background-image value 'radial-gradient(farthest-corner, red, blue)' assert_equals: expected "radial-gradient(rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(red, blue)"
-FAIL Property background-image value 'radial-gradient(farthest-corner at center, red, blue)' assert_equals: expected "radial-gradient(rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(red, blue)"
-FAIL Property background-image value 'radial-gradient(farthest-corner at 50%, red, blue)' assert_equals: expected "radial-gradient(rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(red, blue)"
-FAIL Property background-image value 'radial-gradient(farthest-corner at 10px 10px, red, blue)' assert_equals: expected "radial-gradient(at 10px 10px, rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(at 10px 10px, red, blue)"
+PASS Property background-image value 'radial-gradient(farthest-corner, red, blue)'
+PASS Property background-image value 'radial-gradient(farthest-corner at center, red, blue)'
+PASS Property background-image value 'radial-gradient(farthest-corner at 50%, red, blue)'
+PASS Property background-image value 'radial-gradient(farthest-corner at 10px 10px, red, blue)'
 PASS Property background-image value 'radial-gradient(10px at 20px 30px, rgb(255, 0, 0), rgb(0, 0, 255))'
-FAIL Property background-image value 'radial-gradient(circle calc(-0.5em + 10px) at calc(-1em + 10px) calc(-2em + 10px), red, blue)' assert_equals: expected "radial-gradient(0px at -30px -70px, rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(calc(-0.5em + 10px) at calc(-1em + 10px) calc(-2em + 10px), red, blue)"
-FAIL Property background-image value 'radial-gradient(ellipse calc(-0.5em + 10px) calc(0.5em + 10px) at 20px 30px, red, blue)' assert_equals: expected "radial-gradient(0px 30px at 20px 30px, rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(calc(-0.5em + 10px) calc(0.5em + 10px) at 20px 30px, red, blue)"
-FAIL Property background-image value 'radial-gradient(ellipse calc(0.5em + 10px) calc(-0.5em + 10px) at 20px 30px, red, blue)' assert_equals: expected "radial-gradient(30px 0px at 20px 30px, rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(calc(0.5em + 10px) calc(-0.5em + 10px) at 20px 30px, red, blue)"
+FAIL Property background-image value 'radial-gradient(circle calc(-0.5em + 10px) at calc(-1em + 10px) calc(-2em + 10px), red, blue)' assert_equals: expected "radial-gradient(0px at -30px -70px, rgb(255, 0, 0), rgb(0, 0, 255))" but got "radial-gradient(calc(-0.5em + 10px) at calc(-1em + 10px) calc(-2em + 10px), rgb(255, 0, 0), rgb(0, 0, 255))"
```

Comment:
Surprised cases like this didn’t progress.

Reply:
A follow on when I do the same thing I am doing here for color for the other values in gradients will fix the rest of this.